backend/tests/ohq/test_views.py

```diff
@@ -93,3 +93,76 @@ def test_valid_emails(self):
         content = json.loads(response.content)
         self.assertEqual(1, content["membersAdded"])
         self.assertEqual(1, content["invitesSent"])
+
+
+class QuestionViewTestCase(TestCase):
```

Comment:
I know that I haven't done this, but could you add a docstring here just clearly explain the test case? It should make things a little clearer than just reading the tests